docs/eta-pi-p/automated.ipynb

```diff
@@ -366,34 +324,174 @@
+    "            categorized_sliders_gamma[2].append(slider)\n",
     "\n",
     "    if symbol.name.startswith(\"m_{\"):\n",
     "        slider = w.FloatSlider(\n",
     "            description=Rf\"\\({sp.latex(symbol)}\\)\",\n",
     "            min=0.63,\n",
     "            max=4,\n",
+    "            step=0.01,\n",
     "            value=value,\n",
+    "            continuous_update=False,\n",
     "        )\n",
-    "        mass_sliders.append(slider)\n",
     "        sliders[symbol.name] = slider\n",
+    "        if symbol.name.startswith(\"m_{N\"):\n",
+    "            categorized_sliders_m[0].append(slider)\n",
+    "        elif symbol.name.startswith(R\"m_{\\D\"):\n",
+    "            categorized_sliders_m[1].append(slider)\n",
+    "        elif symbol.name.startswith(\"m_{a\"):\n",
+    "            categorized_sliders_m[2].append(slider)\n",
+    "\n",
+    "    if symbol.name.startswith(\"C_{\"):\n",
+    "        c_latex = sp.latex(symbol)\n",
+    "        phi_latex = c_latex.replace(\"C\", R\"\\phi\", 1)\n",
     "\n",
-    "UI = w.HBox((\n",
-    "    w.VBox(mass_sliders),\n",
-    "    w.VBox(gamma_sliders),\n",
-    "))"
+    "        slider_c = w.FloatSlider(\n",
+    "            description=Rf\"\\({c_latex}\\)\",\n",
+    "            min=0,\n",
+    "            max=10,\n",
+    "            value=abs(value),\n",
+    "            continuous_update=False,\n",
+    "        )\n",
+    "        slider_phi = w.FloatSlider(\n",
+    "            description=Rf\"\\({phi_latex}\\)\",\n",
+    "            min=-np.pi,\n",
+    "            max=+np.pi,\n",
+    "            value=np.angle(value),\n",
+    "            continuous_update=False,\n",
+    "        )\n",
+    "\n",
+    "        sliders[symbol.name] = slider_c\n",
+    "        sliders[symbol.name.replace(\"C\", \"phi\", 1)] = slider_phi\n",
+    "\n",
+    "        cphi_hbox = w.HBox([slider_c, slider_phi])\n",
+    "        if R\"\\D\" in symbol.name:\n",
+    "            categorized_cphi_pair[1].append(cphi_hbox)\n",
+    "        elif \"N\" in symbol.name:\n",
+    "            categorized_cphi_pair[0].append(cphi_hbox)\n",
+    "        elif \"a\" in symbol.name:\n",
+    "            categorized_cphi_pair[2].append(cphi_hbox)\n",
+    "\n",
+    "tab_N_content = w.VBox([\n",
+    "    w.HBox(categorized_sliders_m[0] + categorized_sliders_gamma[0]),\n",
+    "    w.VBox(categorized_cphi_pair[0]),\n",
+    "])\n",
+    "tab_D_content = w.VBox([\n",
+    "    w.HBox(categorized_sliders_m[1] + categorized_sliders_gamma[1]),\n",
+    "    w.VBox(categorized_cphi_pair[1]),\n",
+    "])\n",
+    "tab_a2_content = w.VBox([\n",
+    "    w.HBox(categorized_sliders_m[2] + categorized_sliders_gamma[2]),\n",
+    "    w.VBox(categorized_cphi_pair[2]),\n",
+    "])\n",
+    "\n",
+    "UI = w.Tab([tab_N_content, tab_D_content, tab_a2_content])\n",
+    "UI.set_title(0, \"N*\")\n",
+    "UI.set_title(1, \"Δ*\")\n",
+    "UI.set_title(2, \"a₂*\")"
```

Comment:
ChatGPT. Why not use `titles=["N*", "Δ*", "a₂*"]` as in the other notebook?

Reply:
Also, what is $a_2^*$?

Reply:
https://pdglive.lbl.gov/ParticleGroup.action?init=0&node=MXXX005

because there are two a₂:  (1320) and (1700)